apps/console/src/features/oidc-scopes/components/edit-oidc-scope.tsx

```diff
@@ -140,20 +142,31 @@ export const EditOIDCScope: FunctionComponent<EditScopePropsInterface> = (
         }
     }, [ triggerAddAttributeModal ]);
 
-    const updateOIDCScope = (attributes: string[]): void => {
+    useEffect(() => {
+        if (!tempSelectedAttributes) {
+            return;
+        }
+
+        setUpdatedClaimList(tempSelectedAttributes);
+    }, [ tempSelectedAttributes ]);
+
+
```

Comment:
Let's remove this line.

Reply:
Done in [65dbbb8](https://github.com/wso2/identity-apps/pull/3384/commits/65dbbb875fdb02e7dd27e83530e73767111b8ee6)